projects/fsastorefront-e2e-cypress/cypress.json

```diff
@@ -1,13 +1,13 @@
 {
   "defaultCommandTimeout": 20000,
-  "baseUrl": "https://localhost:4200",
+  "baseUrl": "http://localhost",
   "requestTimeout": 10000,
   "modifyObstructiveCode": false,
   "env": {
     "CLIENT_ID": "financial_trusted",
     "CLIENT_SECRET": "secret",
-    "API_URL": "https://localhost:4200",
-    "API_URL_INT": "https://localhost:4200"
+    "API_URL": "https://10.27.246.242:9012",
+    "API_URL_INT": "https://10.27.241.19:9012"
```

Comment:
Shouldn't this also be localhost? `https://localhost:9002`

Reply:
I can keep localhost, but port must be changed because we are using 9012 on test system.